### PR TITLE
"Develop Locally" document

### PR DESCRIPTION
**Pull request by jwitz**
This document distills the `astro dev` experience into a single doc. I scoped it so that each section represents a high level `astro dev` command (excluding `astro dev ps`). Once the scope of this doc is finalized, I'll go through the rest of the docset and remove redundant information/ link out to this doc accordingly.

### CODE REVIEW

File: docs/develop-locally.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+### Prerequisites
+
+To run Airflow locally via the CLI, you need:
+
+- The Astronomer CLI
+- Docker
+
+### Creating a Local Project
+
+To create a new Airflow project folder, open an empty directory and run `astro dev init`. This command generates the following files in the directory:
```
Comment (paolaperaza) [inline docs/develop-locally.md:22]: > In the empty directory in which you installed the Astronomer CLI, run:
>    ```
>     ./astro dev init
>    ```

Any reason we're not just copy pasting the content we already have on this here? https://beta-docs.astronomer.io/deploy-code

**Review by paolaperaza — CHANGES_REQUESTED** [review]
Hey @jwitz - started to take a pass at this but I think there's a lot we can pull from our existing [Deploy Code doc](https://beta-docs.astronomer.io/deploy-code).

I would:
- Port over Steps 1 + 2 from Deploy Code doc word-for-word
- _Remove_ Steps 1 + 2 from Deploy Code doc
- Copy ["Apply Changes to your Airflow Project"](https://www.astronomer.io/docs/cloud/stable/develop/cli-quickstart#apply-changes-to-your-airflow-project) from Gen1 Docs and replace the current content in `Updating a Local Project`
- Rename `Monitoring a Local Project` to just "Access Local Logs". Monitoring is too heavy for a word in this use case when all we're really giving them is logs
- Link out Airflow CLI, rename `Run Airflow Commands` to `Run Airflow CLI Commands`
- Link out to "Customize your Image" doc obvi as soon as we have that
- Link to Deploy Code at the end

We also don't want to use "Local Project" anywhere here - it's your _Astronomer Project_. You can test it locally, but it's the same project people (can) then push up to Astronomer in Production

**Comment by jwitz** [comment]
@paolaperaza Based on your comments about the doc scope, I updated this PR significantly. It now includes all necessary links to other docs, plus updates to CLI commands for the 1.0.0 version

**Review by paolaperaza — APPROVED** [review]
Looks good @jwitz - some minor comments but will let you handle!

**Pull request merged**